.env

```diff
@@ -0,0 +1 @@
+LUA_PATH="$PWD/src/?.lua;$LUA_PATH"
```

Comment:
I forced it :)
It is good to have it but feel free to remove if you do not like it